check_line_length.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import glob
+import os
+import re
+import sys
+
+LIMIT = 75
+
+
+def Find(string):
+    """
+    Check if there is a url present in the string
+    """
+
+    url = \
+        re.findall('http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\), ]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+                   , string)
+    if url:
+        return True
+    else:
+        return False
+
+
+def limit_line_length(file_path):
+    """
+    Check if the line is in the given limit or not
+    """
+
+    file = file_path.split('/')[-1]
+
+    with open(file_path) as f:
+        lines = f.readlines()
+
+    for (line_number, line) in enumerate(lines, start=1):
+        length = len(line)
+        if length > LIMIT and Find(line) is not True:
+            print('line {} in file {} is longer than {}'.format(line_number,
+                    file, LIMIT))
+            sys.exit(1)
```

Comment:
Why `sys.exit(1)` here? Let's just print all the error lines and then exit. 
Wouldn't that be better?
@cking100 @nemesisdesign

Reply:
1. it is more clean way of doing it
2. The flag we are using in `runsphinx-build' that is `-W` which turn warnings into errors such that the build stops at the first warning and sphinx-build exits with exit status 1 and so take less build time(i think). 
Definitely, need your thoughts @atb00ker @nemesisdesign

Reply:
@cking100 @atb00ker we need to have `sys.exit(1)` to have the script exit with exit code higher than zero, which on linux systems means something went wrong, so the build will stop there, otherwise the build will pass and we won't catch the issue